### DISCUSSION TITLE
Transfer的右半部功能栏，能否添加 react-sortable-tree 这样的拖动组合层级功能

### DISCUSSION THREAD

**Discussion by pxim** [ant-design/ant-design#37217] [General]
### What problem does this feature solve?

1. 可以让 右侧框数据项
![image](https://user-images.githubusercontent.com/5111701/186381248-05146077-177e-44f8-ba05-e44813ab34f4.png)
自由组合 层级关系，
![image](https://user-images.githubusercontent.com/5111701/186381454-2ef242d6-1376-4083-a3f4-f6b34d98f70c.png)
样例：[（https://frontend-collective.github.io/react-sortable-tree/?path=/story/basics--minimal-implementation）](https://frontend-collective.github.io/react-sortable-tree/?path=/story/basics--modify-nodes)

react-sortable-tree：https://github.com/frontend-collective/react-sortable-tree

### What does the proposed API look like?

。

**Comment by zombieJ** [comment]
可以通过自定义能力定制。

**Reply by pxim** [reply]
我试了，解决不了
